crates/turbopack/src/module_options/module_rule.rs

```diff
@@ -44,11 +40,9 @@ impl ModuleRule {
     }
 }
 
-#[turbo_tasks::value_impl]
 impl ModuleRuleVc {
-    #[turbo_tasks::function]
```

Comment:
I wonder if this is 2x, because `ModuleRuleConditionVc::matches` was a task, and this is a task that only depends on that task (and self, but I don't think that counts).

Reply:
Also, `ModuleRuleConditionVc::matches` was recursive.